src/shelly_switch.cpp

```diff
@@ -316,4 +334,52 @@ void ShellySwitch::InputEventHandler(Input::Event ev, bool state) {
   }
 }
 
+void ShellySwitch::AddPowerMeter(uint16_t *iid) {
+  if (out_pm_ == nullptr) return;
+
+  // Power
+  power_char_ = new mgos::hap::UInt16Characteristic(
+      (*iid)++, &kHAPCharacteristic_EveConsumption, 0, 65535, 1,
+      [this](HAPAccessoryServerRef *,
+             const HAPUInt16CharacteristicReadRequest *, uint16_t *value) {
+        auto power = out_pm_->GetPowerW();
+        if (!power.ok())
```

Comment:
please put braces around the body of the if statement, since it's multi-line.
ditto below.

Reply:
It's not multiline, it's a guard clause which returns an error if not ok, if ok value will be set and "no error" will be returned.

Reply:
well, it's on two lines, that's multiline in my book :)
omitting blaces is only ok if entire statement fits on one line
```
if (!energy.ok()) return kHAPError_Busy;
```

here we need braces to remove any possibility of control flow being wrong if statements are added later.
```
if (!energy.ok()) {
  return kHAPError_Busy;  // Operation failed temporarily, retry later.
}
```

Reply:
Ok know what you mean. What do you think, does we need the comment? Without comment we can use a real oneliner.